Tests/UnitTests/Core/Propagator/NavigatorTests.cpp

```diff
@@ -309,103 +303,128 @@ BOOST_AUTO_TEST_CASE(Navigator_status_methods) {
   // (1) Test for inactivity
   //
   // Run without anything present
-  navigator.status(state, stepper);
-  BOOST_CHECK(testNavigatorStateVectors(state.navigation, 0u, 0u, 0u, 0u));
```

Comment:
the ordering of the diff in this file makes it kind of difficult to compare the previous and the new tests, did you add/remove tests?

Reply:
In principle they should all be there, I just had to change the setup a bit because I can't modify the config parameter anymore, since they're `const` in the config member of the navigator, so I have to make new navigators.

Reply:
Ok I see, follow up question: do you think it could be useful to be able to change the navigator configuration, or do you think creating a new navigator is preferable?

Reply:
We do use the `Config` pattern frequently, so, I don't know.